lockbox-ios/View/BiometryOnboardingView.swift

```diff
@@ -0,0 +1,67 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import UIKit
+import RxSwift
+import RxCocoa
+
+class BiometryOnboardingView: UIViewController {
+    internal var presenter: BiometryOnboardingPresenter?
+
+    @IBOutlet internal weak var biometricImage: UIImageView!
+    @IBOutlet internal weak var biometricTitle: UILabel!
+    @IBOutlet internal weak var biometricSubTitle: UILabel!
+    @IBOutlet internal weak var enableButton: UIButton!
+    @IBOutlet internal weak var notNowButton: UIButton!
```

Comment:
Question: why use `weak` and `!`?

Reply:
That's what IB does when you do the option-draggy to get `IBOutlet`s. I'm not sure why they're `weak` and `!` now that you ask ....